translation/en/RedirectedResources/assets/abdata/communication/info_00/optiondisplayitems_02/zz_machineTranslation.txt

```diff
@@ -0,0 +1,34 @@
+OPTION[1]:確かに変わってるね=It's definitely different...
+確かに変わってるね=That's certainly odd.
+OPTION[2]:お好き=I like it
+お好き=I'm not sure what to expect.
+OPTION[3]:育ち盛りですから=Because he's growing up.
+育ち盛りですから=She's growing up.
+OPTION[4]:自炊してる=I do, actually.
+自炊してる=I do, actually.
+OPTION[4]:自炊はちょっと…=I don't cook for myself...
+自炊はちょっと…=Cooking for myself is a bit...
+OPTION[7]:新しい服選び=Choosing new clothes
+新しい服選び=Choosing new clothes
+OPTION[7]:普段は制服だし…=I'm usually in uniform...
+普段は制服だし…=They're usually in uniform...
+OPTION[8]:キミ以上に似合う人はいない=No one looks better than you
+キミ以上に似合う人はいない=The most important thing to remember is that you should not be afraid to ask for help.
```

Comment:
MTL is almost always different for the catch all and the specifically scoped option, and sometimes they're the opposite meaning, other times it's these total garbage sentences like this or `I'm sure you'll be able to find a good deal of information on the web about this.`